lib/smb2.rb

```diff
@@ -2,10 +2,12 @@
 #
 # [[MS-SMB2] Server Mesage Block (SMB) Protocol Versions 2 and 3](https://msdn.microsoft.com/en-us/library/cc246482.aspx)
 module Smb2
-  autoload :VERSION, 'smb2/version'
-  autoload :Packet, 'smb2/packet'
-  autoload :Dispatcher, 'smb2/dispatcher'
   autoload :Client, 'smb2/client'
+  autoload :Dispatcher, 'smb2/dispatcher'
+  autoload :Packet, 'smb2/packet'
+  autoload :Tree, 'smb2/tree'
+  autoload :File, 'smb2/file'
```

Comment:
Please keep the autoloads ordered by the constant name so that it's easier to spot if a constant is missing.